### PR TITLE
Adding Roman Urdu Hate Speech dataset

### PR DESCRIPTION
**Pull request by bp-high**
This Pull request will add the Roman Urdu Hate speech Dataset.

### CODE REVIEW

**Comment by bp-high** [comment]
@lhoestq can you review when you have some time? Also were the previous CI fails due to the Google Drive tests which were excluded by #3982 ?

**Comment by lhoestq** [comment]
> were the previous CI fails due to the Google Drive tests which were excluded by https://github.com/huggingface/datasets/pull/3982 ?

Yes exactly, merging `master` into your branch fixed the CI ;)

File: datasets/roman_urdu_hate_speech/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- crowdsourced
+languages:
+- ur
+licenses:
+- mit
+multilinguality:
+- monolingual
+pretty_name: roman_urdu_hate_speech
+size_categories:
+- unknown
```
Comment (lhoestq) [inline datasets/roman_urdu_hate_speech/README.md:14]:
```suggestion
- 1K<n<10K
```

File: datasets/roman_urdu_hate_speech/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+### Citation Information
+
+@inproceedings{rizwan2020hate,
+  title={Hate-speech and offensive language detection in roman Urdu},
+  author={Rizwan, Hammad and Shakeel, Muhammad Haroon and Karim, Asim},
+  booktitle={Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)},
+  pages={2512--2522},
+  year={2020}
+}
```
Comment (lhoestq) [inline datasets/roman_urdu_hate_speech/README.md:174]:
````suggestion
```bibtex
@inproceedings{rizwan2020hate,
  title={Hate-speech and offensive language detection in roman Urdu},
  author={Rizwan, Hammad and Shakeel, Muhammad Haroon and Karim, Asim},
  booktitle={Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)},
  pages={2512--2522},
  year={2020}
}
```
````

File: datasets/roman_urdu_hate_speech/README.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+0:  Abusive/Offensive
+1:  Normal
+2:  Religious Hate
+3:  Sexism
+4:  Profane/Untargeted
+
+An example from Roman Urdu Hate Speech looks as follows:
+{'tweet':'there are some yahodi daboo like imran chore zakat khore'
+'label':0
+}
```
Comment (lhoestq) [inline datasets/roman_urdu_hate_speech/README.md:92]:
````suggestion
```
{
  'tweet': 'there are some yahodi daboo like imran chore zakat khore'
  'label': 0
}
```
````

File: datasets/roman_urdu_hate_speech/README.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+  title={Hate-speech and offensive language detection in roman Urdu},
+  author={Rizwan, Hammad and Shakeel, Muhammad Haroon and Karim, Asim},
+  booktitle={Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)},
+  pages={2512--2522},
+  year={2020}
+}
+
+### Contributions
+
+Thanks to
```
Comment (lhoestq) [inline datasets/roman_urdu_hate_speech/README.md:178]:
```suggestion
Thanks to [@bp-high](https://github.com/bp-high), for adding this dataset.
```

**Review by lhoestq — APPROVED** [review]
Thanks for adding this dataset ! the dataset card and the dataset script look all good :)

**Pull request merged**